yt_dlp/extractor/youtube/_video.py

```diff
@@ -3243,6 +3243,10 @@ def append_client(*client_names):
                 # web_creator may work around age-verification for all videos but requires PO token
                 append_client('tv_embedded', 'web_creator')
 
+            status = traverse_obj(pr, ('playabilityStatus', 'status'))
+            if status not in ('OK', 'LIVE_STREAM_OFFLINE', 'AGE_CHECK_REQUIRED', 'AGE_VERIFICATION_REQUIRED'):
+                self.write_debug(f'{video_id}: {client} player response playability status: {status}')
```

Comment:
Imo this should be a warning

Reply:
mosttoo verbose imo. we already have a warning when all prs have a bad playability status.